.env.example

```diff
@@ -5,6 +5,7 @@ APP_DEBUG=true
 APP_URL=http://escooters.blumilk.localhost
 
 MAPBOX_TOKEN=
+MAPBOX_URL=https://api.mapbox.com/geocoding/v5/mapbox.places/
```

Comment:
It will be better to left here `https://api.mapbox.com/geocoding/v5` only. It will allow to use other endpoints than `mapbox.places`